configure.ac

```diff
@@ -1283,13 +1368,21 @@ then
     else
         AC_SUBST(UNROLL_LOOPS,0)
     fi
+    # POP CFLAGS
     CFLAGS="$save_CFLAGS"
 else
+    # POP CFLAGS
     CFLAGS="$save_CFLAGS"
     AX_CHECK_COMPILE_FLAG([$CFLAGS],[],AC_MSG_ERROR(["Couldn't compile with given CFLAGS!"]))
     AC_SUBST(UNROLL_LOOPS,0)
 fi
 
+# Trim flags
+LIBS=`echo "$LIBS" | tr ' ' '\n' | sort | uniq | xargs`
+LDFLAGS=`echo "$LDFLAGS" | tr ' ' '\n' | sort | uniq | xargs`
+CFLAGS=`echo "$CFLAGS" | tr ' ' '\n' | sort | uniq | xargs`
```

Comment:
There are options that must come one after the other. For eg:
1. `-I /opt/foo/include`  (note the space between)
2. `-Xclang -target=x86_64-linux-gnu`
3. `-Wl,-rpath -Wl,/opt/foo/lib`